### PR TITLE
Add PR and issue templates

### LINKED ISSUE #34: Add PR and issue templates
### Description

Add template for PR and issue for github

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #34

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'bug'
+assignees: ''
+---
+
+### Description
+Please describe the bug and provide suggestions how to fix it, if possible.
```
Comment (TalhaMaliktz) [inline .github/ISSUE_TEMPLATE/bug-report.md]:
```suggestion
Please describe the bug and suggest how to fix it, if possible.
```

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+### Location
+Please describe where the content should be fixed / updated
```
Comment (TalhaMaliktz) [inline .github/ISSUE_TEMPLATE/bug-report.md]:
```suggestion
Please describe where the content should be fixed/updated.

### Additional Information
```

File: .github/ISSUE_TEMPLATE/docs-update.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+name: Documentation update
+about: Propose changes in the documentation
+title: ''
+labels: 'documentation'
+assignees: ''
+---
+
+### Description
+
+Please describe what documentation updates are needed
```
Comment (TalhaMaliktz) [inline .github/ISSUE_TEMPLATE/docs-update.md]:
```suggestion
Please describe what documentation updates are needed.
```

File: .github/ISSUE_TEMPLATE/docs-update.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+### Location
+
+Please describe where the content should be added / updated
```
Comment (TalhaMaliktz) [inline .github/ISSUE_TEMPLATE/docs-update.md]:
```suggestion
Please describe where the content should be added/updated.
```

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+name: Feature request
+about: Propose a new feature to the documentation
+title: ''
+labels: 'enhancement'
+assignees: ''
+---
+
+### Description
+
+Please describe what functionality is needed
```
Comment (TalhaMaliktz) [inline .github/ISSUE_TEMPLATE/feature-request.md]:
```suggestion
Please describe what functionality is needed.
```

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+### Motivation
+
+Please describe why it is needed
```
Comment (TalhaMaliktz) [inline .github/ISSUE_TEMPLATE/feature-request.md]:
```suggestion
Please describe why it is needed.
```

**Review by TalhaMaliktz — CHANGES_REQUESTED** [review]
Please consider the following!

**Review by TalhaMaliktz — APPROVED** [review]
LGTM 🎉

**Review by shuse2 — APPROVED** [review]

**Pull request merged**